clients/monitoring/series.ts

```diff
@@ -97,31 +100,35 @@ export class DailyUptimeRatioConverter {
     this._days[key] = currDay;
   }
 
-  /**
-   *  For a given date, returns the Date of the next UTC day at 00:00:00
-   *  ex: 19/07/2021 14:35:00 UTC => 20/07/2021 00:00:00
-   *  */
-  getUTCNextDay = (d: Date): Date => {
-    const next = new Date(
-      Date.UTC(d.getUTCFullYear(), d.getUTCMonth(), d.getUTCDate() + 1)
-    );
-    return next;
-  };
-
-  addUTCDay = (d: Date, day: number) => {
-    const logDate = new Date(d);
-    logDate.setUTCDate(logDate.getUTCDate() + day);
-    return logDate;
-  };
-
   export(): IRatio[] {
     return Object.values(this._days)
       .sort((a, b) => (a.date < b.date ? -1 : 1))
       .map((day) => {
         return {
           ratioNumber: day.uptime,
-          date: this.getDDMMYYYY(day.date),
+          date: getDDMMYYYY(day.date),
         };
       });
   }
 }
+
+function getDDMMYYYY(d: Date): string {
+  return new Intl.DateTimeFormat('fr-FR', { timeZone: 'UTC' }).format(d);
+}
+
+function addUTCDay(d: Date, day: number) {
+  const logDate = new Date(d);
+  logDate.setUTCDate(logDate.getUTCDate() + day);
+  return logDate;
+}
+
+/**
+ *  For a given date, returns the Date of the next UTC day at 00:00:00
+ *  ex: 19/07/2021 14:35:00 UTC => 20/07/2021 00:00:00
+ *  */
+function getUTCNextDay(d: Date): Date {
+  const next = new Date(
+    Date.UTC(d.getUTCFullYear(), d.getUTCMonth(), d.getUTCDate() + 1)
+  );
+  return next;
+}
```

Comment:
I moved those pure functions outside class scope for readiness (we know they don't depend on class internal that way).